CHANGELOG.md

```diff
@@ -16,7 +16,8 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - The AWS detector now adds the cloud zone, host image ID, host type, and host name to the returned `Resource`. (#410)
 - Add Amazon ECS Resource Detector for AWS X-Ray. (#466)
 - Add propagator for AWS X-Ray (#462)
-  
+- A new `gcp.CloudRun` detector for detecting resource from a Cloud Run instance. (#455)
```

Comment:
This needs to be moved the `Unreleased` section.

Reply:
Moved to Unreleased.